File: Hello2.java
public lass Person {
    privte String name;
    privte int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public void displayInfo() {
        System.out.println("Name: " + name + ", Age: " + age);
    }

    public static vid main(String[] args) {
        Person person = new Person("Alice", 25);
        person.displayInfo();
    }
}

File: Hello2.py
lass Person:
    ef __init__(self, name, age):
        self.name = name
        self.age = age

    def display_info(self):
        print(f"Name: {self.name}, Age: {self.age}")

if __name__ == "__main__":
    person = Person("Alice", 25)
    person.display_info()
